react/components/molecules/avatar-list/avatar-list.js

```diff
@@ -0,0 +1,138 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+import { Avatar, Text } from "../../atoms";
+
+export class AvatarList extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            avatars: PropTypes.arrayOf(PropTypes.string).isRequired,
+            size: PropTypes.number,
+            avatarProps: PropTypes.object,
+            visibleAvatars: PropTypes.number,
+            style: ViewPropTypes.style,
+            textStyle: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            label: undefined,
+            size: 40,
+            avatarProps: {},
+            visibleAvatars: 3,
+            style: {}
+        };
+    }
+
+    _avatars() {
+        if (this.props.avatars.length <= this.props.visibleAvatars) return this.props.avatars;
+        const reducedAvatars = this.props.avatars.slice(0, this.props.visibleAvatars + 1);
+        return reducedAvatars;

```

Comment:
I would either return this directly like `return this.props.avatars.slice...` or rename it to `slicedAvatars` or `trimmedAvatars` to avoid "reduce" not to confuse with Array.reduce.

Alternatively, it could just be the ternary:

```js
this.props.avatars.length <= this.props.visibleAvatars ? this.props.avatars : this.props.avatars.slice(0, this.props.visibleAvatars + 1)
```